posthog/api/feature_flag.py

```diff
@@ -201,21 +208,40 @@ def my_flags(self, request: request.Request, **kwargs):
 
     @action(methods=["GET"], url_path="activity", detail=False)
     def all_activity(self, request: request.Request, **kwargs):
-        activity = load_activity(scope="FeatureFlag", team_id=self.team_id)
-        return Response(
-            {"results": ActivityLogSerializer(activity, many=True,).data, "next": None, "previous": None,},
-            status=status.HTTP_200_OK,
-        )
+        limit = int(request.query_params.get("limit", "10"))
+        page = int(request.query_params.get("page", "1"))
```

Comment:
It's more typical to use `offset` with `limit` rather than `page`. With `page` it should be more like `page_size`. 🤔 Thinking about what pattern is best here

Reply:
DRF supports both offset/limit and page/limit. 

But using page meant I could use the django paginator and not do any maths at either end. URL, Pagination Control and API all use the same value.

Reply:
https://www.django-rest-framework.org/api-guide/pagination/ for the DRF link

I'd say there's not a one-right-way™️ :) and that `page` is understandable...